### DISCUSSION TITLE
How do I determine the type of the Image?

### DISCUSSION THREAD

**Discussion by yimuu** [TorchIO-project/torchio#821] [Q&A]
When I got subjects form the dataset, some of the images are ByteTensor and some are FloatTensor.
![image](https://user-images.githubusercontent.com/51940981/153696424-4490a2d3-8c58-4705-ac6b-45ebec1bfd63.png)

**Comment by fepegar** [comment]
Hi, @yimuu. Could you please ellaborate? I'm not sure what you're asking.
